piker/ui/_interaction.py

```diff
@@ -295,6 +301,18 @@ def mouseDragEvent(self, ev, axis=None):
         # Scale or translate based on mouse button
         if ev.button() & (QtCore.Qt.LeftButton | QtCore.Qt.MidButton):
 
+            # zoom only y-axis when click-n-drag on it
```

Comment:
This is the core of it.
If a user click-drags on the y-axis we do scaling on the y-dimension only.